models/index.js

```diff
@@ -0,0 +1,43 @@
+'use strict'
+
+const fs = require('fs')
+const path = require('path')
+const Sequelize = require('sequelize')
+const process = require('process')
+const basename = path.basename(__filename)
+const env = process.env.NODE_ENV || 'development'
+const db = {}
+
+let sequelize = new Sequelize({
+  username: env.MYSQL_USER,
+  password: env.MYSQL_PASSWORD,
+  database: env.MYSQL_DATABASE,
+  host: "localhost",
```

Comment:
```suggestion
  host: env.MYSQL_HOST,
```
changer l'host de la config sequelize

Reply:
[6e3755d](https://github.com/FELS-dev/backend/pull/2/commits/6e3755d48a3b139b3e8ef2627581e331f7143f31)